cmodel/predefined/_seirv_fixed.py

```diff
@@ -0,0 +1,25 @@
+from typing import List
+
+from cmodel.model import CompartmentalModel
+
+
+class SEIR_model(CompartmentalModel):
```

Comment:
Change class name to one similar to ``cmodel.predefined._seirv_model.ModelSEIRV``

```suggestion
class ModelSimpleSEIRV(CompartmentalModel):
```